src/zypper-plugins/enabled_repos_upload.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from os import environ
+import sys
+import logging, traceback
+
+from katello.repos import upload_enabled_repos_report
+from katello.enabled_report import EnabledReport
+from subscription_manager.repolib import ZypperRepoFile
+
+from zypp_plugin import Plugin
+
+class KatelloEnabledUploadZyppPlugin(Plugin):
```

Comment:
class name seems a bit ... ambiguous? How about EnabledReposUploadPlugin it's already in the zypper-plugins module so we know it's for zypper. Picky, I know.